libs/utils/utils.js

```diff
@@ -335,13 +335,24 @@ export async function loadBlock(block) {
 
 export function decorateSVG(a) {
   const { textContent, href } = a;
-  const ext = textContent?.substr(textContent.lastIndexOf('.') + 1);
+  const altTextFlagIndex =  textContent.indexOf('|');
+  const sanitizeTextContent = altTextFlagIndex === -1
```

Comment:
Usually verbs are for functions. Would `sanitizedTextContent` be more appropriate?